src/theme/states/states.tokens.ts

```diff
@@ -0,0 +1,70 @@
+import globals from 'theme/globals';
+import semantic from 'theme/tokens/semantic';
+
+type ComponentVariant =
+  | 'primary'
+  | 'secondary'
+  | 'tertiary'
+  | 'danger'
+  | 'inverted'
+  | 'invertedAlt';
+
+export type StatesTokens = {
+  focus: {
+    outline: string;
+  };
+  disabled: { opacity: string; cursor: string };
+  inactive: {
+    backgroundColor: Record<ComponentVariant, string>;
+  };
+  active: {
+    backgroundColor: Record<ComponentVariant, string>;
+  };
+  hover: {
+    backgroundColor: Record<ComponentVariant, string>;
+  };
+};
+
+const tokens: StatesTokens = {
+  focus: {
+    outline: `${semantic.borderColor.get('interactive.focused')} auto ${globals.borderWidth.get(
```

Comment:
i saw that this was used in other files but as stated before you have to pass the `theme` here and pick "semantic" from `theme.tokens`. This way it will use the **active** theme for getting the value if it is **semantic** or **dark** etc

Reply:
yeap, you are totally right, I will change all instances of semantic across the project